src/main/java/swbot/Event.java

```diff
@@ -0,0 +1,74 @@
+package swbot;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * A type of task that lets the user create an event to plan for. An event task
+ * has a start date and an end date along with a description of what the event is.
+ */
+public class Event extends Task {
+    protected LocalDateTime startDate;
+    protected LocalDateTime endDate;
+
+    /**
+     * Creates an event task with description, startDate and an endDate
+     *
+     * @param description describes the task to be done
+     * @param startDate starting date of the event
+     * @param endDate when the event ends
+     * @throws BuzzException if the formatting of the date/time is wrong
+     */
+    public Event(String description, String startDate, String endDate) throws BuzzException {
+        super(description);
+        try {
+            DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
```

Comment:
You could consider another, more apt variable name; just like @wes-w-z-h pointed out :)